prymer/api/variant_lookup.py

```diff
@@ -254,6 +250,30 @@ def __init__(
         self.min_maf: Optional[float] = min_maf
         self.include_missing_mafs: bool = include_missing_mafs
 
+    @classmethod
+    def build(
+        cls,
+        vcf_paths: list[Path],
+        use_cache: Optional[bool] = True,
+        min_maf: Optional[float] = 0.00,
+        include_missing_mafs: Optional[bool] = False,
```

Comment:
**suggestion** I don't think any of these need to be `Optional`. 

Two general rules of thumb - 

1. If you have a default value that isn't `None`, you probably don't need to make the parameter `Optional`
2. You shouldn't ever have an `Optional[bool]` - this gets buggy and tricky pretty quickly

🙂 

```suggestion
        use_cache: bool = True,
        min_maf: float = 0.0,
        include_missing_mafs: bool = False,
```

Reply:
Agreed that these should be non-Optional.

Though I might argue in fact that `include_missing_mafs` should remain `Optional` and be `None` by default, with the behavior being that if `min_maf <= 0` it gets set to True, and otherwise defaults to False.

If we don't do that, then I think it should default to True - i.e. include all the variants in the VCFs until told otherwise.